sdk/core/pom.service.xml

```diff
@@ -9,6 +9,13 @@
   <packaging>pom</packaging>
   <version>1.0.0</version><!-- Need not change for every release-->
   <modules>
-    <module>../../pom.client.xml</module>
+    <module>azure-core</module>
```

Comment:
/CC @weshaggard 
The reason this was changed to pom.client.xml was to ensure that when core built everything else built. Swapping this back to just the list of core libraries is going to revert that and we're probably going to want to have whatever the "build everything for client" equivalent is going to be.

Reply:
Correct. We needed a way to have the core pipeline build and test everything that depended on core. If you can come up with another clever way to do that we can switch otherwise I'd keep this the way it is.

Reply:
Yeah so I created this PR knowing this problem wasn't solved with the hope that it would spur some conversation about the issues created by the way we have this setup and other approaches that we could possibly take.

Reply:
Coming back to this. The way to make this build process the downstream dependents is to use the ```-amd``` switch.